### ISSUE TITLE
hanging other scripts

### ISSUE THREAD

**Issue by ShadowOfVoid** [TanguyOrtegat/gcphone-from-GKSKnight#12]
i managed to get it to work, but when everyone close the phone and use another script like inventory or even chat , that resource will get stuck and i should restart it to get it to work again, its so annoying

**Comment by TanguyOrtegat** [comment]
that is weird because does work flawwless on my server

**Comment by ShadowOfVoid** [comment]
yes it works fine , but i have 2 problem with it , first is for the first time when you use it , it hangs like 5 sec then it works without a problem , then every time you close it open chat immediately then use chat or even inventory for 10 sec , then it will hang and not responding till you restart that resource , i dont know what should i do

**Comment by TanguyOrtegat** [comment]
weird don't have that issue, neither others, so maybe incompatible script / code?

**Comment by killerrichard** [comment]
same issue ^^

**Comment by majidmazaheri** [comment]
same issue :(

**Comment by majidmazaheri** [comment]
does anybody have solution for this issue?
